src/families/bitcoin/wallet-btc/explorer/index.ts

```diff
@@ -207,28 +207,29 @@ class BitcoinLikeExplorer extends EventEmitter implements IExplorer {
 
     // TODO add a test for failure (at the sync level)
     const client = await this.client.acquire();
-    const res = (
-      await client.client.get(url, {
-        params,
-        // some altcoin may have outputs with values > MAX_SAFE_INTEGER
-        transformResponse: (string) =>
-          // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          JSONBigNumber.parse(string, (key: string, value: any) => {
-            if (BigNumber.isBigNumber(value)) {
-              if (key === "value") {
-                return value.toString();
+    let res: { txs: TX[] } = { txs: [] };
```

Comment:
why change in this file? 🤔

Reply:
Rebased and the change disappeared, should be fine now

Reply:
Rebased and the change disappeared, should be fine now